### ISSUE TITLE
[REQUEST] delete json backup file in settings

### ISSUE THREAD

**Issue by dtantono** [Paperback-iOS/app#546]
Hi 
I know in settings -> backups -> new backup. we can export and create a backup. But, how to delete the .json backup file?

**Comment by Jxfe** [comment]
If you swipe left on the backup itll delete it in the App. You would have to delete it from other locations if you saved it elsewhere.

**Comment by XenitXTD** [comment]
looked up on the paperback.moe website and found the following

https://www.youtube.com/watch?v=5EPN9FORQ1g video sample like who ever would think this is how it works instead of with a import button

friend had the issue found it here https://paperback.moe/help/faq/#adjust-content-settings

![image](https://user-images.githubusercontent.com/5268461/164214061-2f334b85-20ec-447c-a170-d15828f15917.png)

Its old hope this helps. for now until they make this a simple import button.

**Comment by dtantono** [comment]
I tried that. But, there is no copy to paperback button or save to paperback with Notes app.

**Comment by XenitXTD** [comment]
Friend had a similar issue with a file saved to onedrive...

He copied the file to iCloud and then did it and it worked... Not sure if this will work for you, try saving it from notes to icloud and uploading.

I assume the app is up to date.

**Comment by dtantono** [comment]
Yes, technically it works. But, I hope they can simplify the restore and backup procedure like tachiyomi with just one tap.

**Comment by celarye** [comment]
As part of a repository-wide cleanup for the 0.9 release, we are resetting our issue tracker to more efficiently track new feedback and feature requests. If this issue is still relevant or critical to your workflow, please feel free to reopen it with updated context. Thank you for your support!
